workspace/events/lib/class.emailevent.php

```diff
@@ -39,6 +39,11 @@ protected function getSenderEmail()
         return $this->getFromEmail();
     }
     
+    public function getReplyTo()
+    {
+        return null;
```

Comment:
Should this be configurable in config.php and not here ?

Reply:
No: this would break the default. Also, this class is abstract, and needs to be extented, so you'll be able to do what you want !

Reply:
Alright !